### ISSUE TITLE
A11y_Visual Studio Code_HomePage_Resize Text: At 150-175% zoom level, 'Version 1.55 is now available' banner is not visible.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4513]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#WCAG1.4.4#Resize Text#Zoom#Visual Studio Code Services#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Note:
This issue is observed through out the application

### Repro Steps:

1. Zoom the page to 150-175%
2. Hit [Visual Studio Code](https://code.visualstudio.com/) link
3. Navigate to download button in top navigation section and press tab
4. Verify that 'Version 1.55 is now available' banner is not visible at 150-175% zoom level

### Actual Result:
At 150-175% zoom level, 'Version 1.55 is now available' banner is not visible.

### Expected Result:
At 150-175% zoom level, 'Version 1.55 is now available' banner should be clearly visible. From 100-200% zoom levels, entire content in the page should be clearly visible. There should be no loss of content in the page

### User Impact:
Users with low vision who zoom the page for better visibility will not know about the new versions which are available.

### Attachment:

https://user-images.githubusercontent.com/83570638/116966136-1c6b4f80-accd-11eb-8f3d-968c6f76eb32.mp4

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified issue in Below environment:
URL: https://code.visualstudio.com/
Issue is Fixed
![image](https://user-images.githubusercontent.com/100189407/217513946-b5680e5c-e5f6-4105-8f3a-c9cc2dc93aa8.png)
